extensions/wikia/Wall/WallMessage.class.php

```diff
@@ -1548,34 +1415,25 @@ public function getPostedAsBot() {
 	public function setInCommentsIndex( $prop, $value, $useMaster = false ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$useMaster". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)